src/planscape/collaboration/utils.py

```diff
@@ -0,0 +1,110 @@
+from collaboration.models import Collaborator, Permissions, Role
+from planning.models import PlanningArea, Scenario
+from django.contrib.auth import get_user_model
+from django.contrib.contenttypes.models import ContentType
+
+
+User = get_user_model()
+
+
+def is_creator(user: User, planning_area: PlanningArea):
+    return planning_area.user.pk == user.pk
+
+
+def has_access(user_id, model, permission):
+    try:
+        content_type = ContentType.objects.get_for_model(model)
+        entry = Collaborator.objects.get(
+            collaborator_id=user_id,
+            content_type_id=content_type,
```

Comment:
does this need to be content_type_id = content_type.id? or is this working?

Reply:
its working but probably should be `content_type = content_type` ?